### PR TITLE
Add contributing page.

### PR DESCRIPTION
**Pull request by brendandahl**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+| chrome_ref     |          | Corresponding 'id' property from https://www.chromestatus.com/features.json                                                                                                                            |
+| ie_ref         |          | Corresponding 'name' property from https://raw.githubusercontent.com/MicrosoftEdge/Status/production/app/static/ie-status.json                                                                         |
+
+## Should We Add Feature X?
+
+The feature makes a good candidate if any of the following are true:
+
+* a standardization process has started
+* major browsers have started implementing it
+* it been requested by web designers/developers
```
Comment (marco-c) [inline CONTRIBUTING.md]: Verb missing.

**Comment by digitarald** [comment]
@brendandahl r+ with fixed nits

**Comment by mykmelez** [comment]
> r? @mykmelez as he had some ideas on how to phrase our expectations for features

It's a reasonable start. It's hard to design such criteria in advance, as they'll need trial by fire. We can derive to some extent from the features we've already chosen to include, but the proof will be in the pudding of pull requests from strangers.

I might add a note about the criteria not being definitive. We say "good candidate", which implies that, but it'd be helpful to make it explicit, since we won't add every feature that meets one of these criteria (especially the third one).

I might add "(or experimenting with it)" to the second line, as some "implementations" are more experimental (thinking about the WebUSB API, although https://www.chromestatus.com/feature/5651917954875392 elides the distinction).

Finally, I might add some anti-criteria; especially if we have a list of features that we decided *not* to include, from which we can derive them; and then an example for each criterion.

**Comment by digitarald** [comment]
We should link to https://github.com/mozilla/platatus/new/master/features as the *quickest* way to create feature pull requests.

**Comment by wfwalker** [comment]
+1 to @digitarald 's last comment, I did all my feature PR's that way

**Comment by digitarald** [comment]
Merging for now, we can tweak it easily within the repo.

**Pull request merged**